client/components/jetpack/card/jetpack-product-slide-out-card/index.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { useTranslate, TranslateResult } from 'i18n-calypso';
+import React, { ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button, ProductIcon } from '@automattic/components';
+import { preventWidows } from 'calypso/lib/formatting';
+import PlanPrice from 'calypso/my-sites/plan-price';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+type Props = {
+	className?: string;
+	iconSlug: string;
+	productName: TranslateResult;
+	description?: ReactNode;
+	currencyCode: string | null;
+	price: number;
+	billingTimeFrame: TranslateResult;
+	buttonLabel: TranslateResult;
+	buttonPrimary: boolean;
```

Comment:
Just a nit pick, but maybe should we make this an optional prop?  And give it a default value?

Reply:
Good idea!